services/ui-src/src/components/pages/Dashboard/DashboardTable.tsx

```diff
@@ -57,9 +57,17 @@ export const HorizontalTable = (
         <Tr key={report.id}>
           {showEditNameColumn && (
             <Td fontWeight={"bold"}>
-              <button onClick={() => openAddEditReportModal(report)}>
-                <Image src={editIcon} alt="Edit Report Name" minW={"1.75rem"} />
-              </button>
+              {report.status !== ReportStatus.SUBMITTED ? (
+                <button onClick={() => openAddEditReportModal(report)}>
+                  <Image
+                    src={editIcon}
+                    alt="Edit Report Name"
+                    minW={"1.75rem"}
+                  />
+                </button>
+              ) : (
+                <div />
+              )}
```

Comment:
Nit: I'd like to see nothing more than an empty div here
```suggestion
              ) : null
              }
```